skripte/sql_komplet.sql

```diff
@@ -0,0 +1,41 @@
+CREATE DATABASE IF NOT EXISTS restoran;
+USE restoran;
+
+-- Tablica Konobar
+CREATE TABLE Konobar (
+    KonobarId INT AUTO_INCREMENT PRIMARY KEY,
+    Ime VARCHAR(50),
+    Prezime VARCHAR(50)
+);
+
+-- Tablica Stol
+CREATE TABLE Stol (
+    StolId INT AUTO_INCREMENT PRIMARY KEY,
+    KonobarId INT,
```

Comment:
ovo je možda višak jer nigdje nemamo eksplicitno navedeno da je 1 konobar zadužen za 1 stol no to su detalji. sve ostalo je odlično!